src/gui/properties/speedplotview.cpp

```diff
@@ -169,6 +170,63 @@ int SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    //   table of supposed nice steps for grid marks to get nice looking quarters of scale
```

Comment:
@dzmat,  you don't seem familiar with the qBittorrent Coding Style. Maybe now is the time to get acquainted with it? I can't point to every obvious violation when most of the code below is broken.